pyinfra/facts/pipx.py

```diff
@@ -0,0 +1,81 @@
+import re
+
+from pyinfra.api import FactBase
+
+from .util.packaging import parse_packages
+
+
+# TODO: move to an utils file
+def parse_environment(output):
+    environment_REGEX = r"^(?P<key>[A-Z_]+)=(?P<value>.*)$"
+    environment_variables = {}
+
+    for line in output:
+        matches = re.match(environment_REGEX, line)
+
+        if matches:
+            environment_variables[matches.group("key")] = matches.group("value")
+
+    return environment_variables
+
+
+pipx_REGEX = r"^([a-zA-Z0-9_\-\+\.]+)\s+([0-9\.]+[a-z0-9\-]*)$"
+
+
+class PipxPackages(FactBase):
+    """
+    Returns a dict of installed pipx packages:
+
+    .. code:: python
+
+        {
+            "package_name": ["version"],
+        }
+    """
+
+    default = dict
+    pipx_command = "pipx"
+
+    def requires_command(self, pipx=None):
+        return pipx or self.pipx_command
+
+    def command(self, pipx=None):
```

Comment:
I think we can just always use `pipx` as the command, there's no case where we'd have multiple versions alongside each other I think?